### ISSUE TITLE
[5.x]: Unable to select User Photo for user

### ISSUE THREAD

**Issue by wbrowar** [craftcms/cms#14668]
### Description
I'm unable to select a photo for a user profile, using the **Upload a photo** button.

### Steps to reproduce

1. Create a filesystem (in my case I created a local filesystem)
2. Create an asset volume and select the new filesystem
3. Go to `Settings > User Settings > Settings` and select the volume for the **User Photo Location** field and hit **Save**
4. Go to a user profile and click on **Upload a photo**

### Expected behavior
I expect the asset picker to appear and let me select a photo (or upload a new photo).

### Actual behavior
The button doesn't respond to the mouse click.

### Craft CMS version

5.0.0-beta.11

### PHP version

8.2.12

### Operating system and version

macOS 14.4 (Sonoma)

### Database type and version

MySQL 8.0.32

### Image driver and version

Imagick 3.7.0 (ImageMagick 7.1.1-13)

### Installed plugins and versions

- No plugins, but I have the following modules installed:

- codeeditor	nystudio107\codeeditor\CodeEditor
- closure	nystudio107\closure\Closure
- debug	craft\debug\Module

**Comment by wbrowar** [comment]
NOTE: I tried this without any images uploaded to the asset volume, then again with an image uploaded and it didn't work in either situation.

**Comment by brandonkelly** [comment]
Working on my end. Are you getting any JavaScript errors on the page?

**Comment by wbrowar** [comment]
Ah, okay, so I was testing things out in Chrome and Firefox and noticed it was working in Firefox, but not in Chrome. I went to check the version number but saw that there was an update available, so I did that first and now it's fine.. So for some reason it wasn't working in the older version of Chrome. I should have grabbed that version number but if I had to guess it was maybe behind by 2-3 updates.

It's working now so I'm going to close this ticket.
